common/beerocks/bcl/include/bcl/network/network_utils.h

```diff
@@ -94,6 +94,8 @@ class network_utils {
     static int get_iface_info(network_utils::iface_info &info, const std::string &iface_name);
     static bool get_raw_iface_info(const std::string &iface_name, raw_iface_info &info);
 
+    static std::unordered_map<std::string, std::string> &get_arp_table(bool mac_as_key = true);
```

Comment:
Added a Doxygen to the function on the h file.
I have decided to use string since I wanted to have the flexibility to replace between the MAC and IP on the map object.